src/components/OpenFileDialog/FileList.js

```diff
@@ -32,9 +32,17 @@ export const FileList = ({ type, data, onSelect }) => (
 )
 
 FileList.propTypes = {
-    data: PropTypes.array,
-    type: PropTypes.string,
-    onSelect: PropTypes.func,
+    data: PropTypes.arrayOf(
+        PropTypes.shape({
+            created: PropTypes.string,
```

Comment:
Should these be `isRequired` too?

Reply:
I was thinking about that, I think it should be safe since they are listed in the query's fields, so they should always be there.

Reply:
Fixed.